### PR TITLE
feat(*): ap design sync - I57

### PR DESCRIPTION
**Pull request by jolanglinais**
# Issue #57
Changing styling based on props to fit non-branding and AP-branding

### Changes
- Remove unnecessary Semantic-UI-React `Card`'s
- Follow a system for styling props to pass through to Cicero-UI

### Flags
- Possible styling issue with the Contract Editor's scrollbar when extending past the bottom of the page

### Related Issues
- Issue [MD35](https://github.com/accordproject/markdown-editor/pull/35), [CUI72](https://github.com/accordproject/cicero-ui/issues/72)
- Pull Request [MD66](https://github.com/accordproject/markdown-editor/pull/66), [CUI73](https://github.com/accordproject/cicero-ui/pull/73)

### CODE REVIEW

File: src/components/TextButton.js [hunk 1]
```diff
@@ -1,26 +1,28 @@
 import styled from 'styled-components';
 
+import { AP_THEME } from '../containers/App/themeConstants';
+
 const TextButton = styled.button`
   margin: 0;
   padding: 0;
   border: 0;
   background: transparent;
   font-size: ${props => (props.fontSize ? props.fontSize : '#inherit')};
```
Comment (DianaLease) [inline src/components/TextButton.js]: the `inherit` value should not have a `#`

File: src/containers/App/themeConstants.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+export const LEFT_NAV = {
+  HEADING: AP_THEME.GRAY,
+};
+
+export const CONTRACT_EDITOR = {
+  BUTTON_BACKGROUND_INACTIVE: AP_THEME.DARK_BLUE_LIGHT,
+  BUTTON_BACKGROUND_ACTIVE: '#364C77',
+  BUTTON_BACKGROUND_HOVER: '#364C77',
+  BUTTON_SYMBOL_INACTIVE: '#949CA2', // #747B8F
```
Comment (DianaLease) [inline src/containers/App/themeConstants.js]: not sure what the commented hex value is for

**Review by DianaLease — CHANGES_REQUESTED** [review]
Again, just a couple small comments!

File: src/components/TextButton.js (hunk 1 above)
Comment (jolanglinais) [reply src/components/TextButton.js]: Caught yourself! https://github.com/accordproject/template-studio-v2/commit/30c061d81042410296a1852920a141bb4867b6af#diff-80fa4b1ac52713917acfa4eedc30ee5eR8

File: src/containers/App/themeConstants.js (hunk 2 above)
Comment (jolanglinais) [reply src/containers/App/themeConstants.js]: Ah yes, residual from setting this all up.

**Review by DianaLease — DISMISSED** [review]

**Pull request merged**